src/link/link_info/bond.rs

```diff
@@ -130,6 +138,49 @@ impl<'a, T: AsRef<[u8]> + ?Sized> Parseable<NlaBuffer<&'a T>> for BondAdInfo {
     }
 }
 
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+#[non_exhaustive]
+pub enum BondMode {
+    RoundRobin,
+    ActiveBackup,
+    Xor,
+    Broadcast,
+    Ieee8023Ad,
```

Comment:
Took the liberty of adding `Ieee` since an identifier can't start with a digit

Reply:
OK to me.